package.json

```diff
@@ -58,6 +57,7 @@
     "express-session": "1.17.3",
     "helmet": "4.6.0",
     "https-proxy-agent": "5.0.1",
+    "jose": "4.14.4",
```

Comment:
<img src="https://media1.giphy.com/media/O0hKo5TyzRnldNiniu/giphy.gif"/>

Reply:
Vi har funnet fram til noen lovende tilpasninger i docker-compose, Anders skal sjekke det ut i dag.

Reply:
Holder på å prøve det ut i syfomoteoversikt, men foreløpig ikke i mål så vi bør nok vente med å merge denne og tilsvarende for syfomodiaperson til vi har et lokalt oppsett som funker.

Reply:
Ser ut som vi finner en løsning på det, så merger denne nå.